keras/backend/tensorflow_backend.py

```diff
@@ -20,6 +20,9 @@
 from .common import set_image_dim_ordering
 from .common import image_dim_ordering
 
+import logging
```

Comment:
can tf.logging be hooked into this? that may be the best course of action https://www.tensorflow.org/api_docs/python/tf/logging